library/Backends/Michelson/src/Juvix/Backends/Michelson/DSL/InstructionsEff.hs

```diff
@@ -193,6 +193,42 @@ nameSymb :: Env.Reduction m => NameSymbol.T -> Types.RawTerm -> m Env.Expanded
 nameSymb symb f@(Types.Ann usage _ _) =
   inst f <* modify @"stack" (VStack.nameTop symb usage)
 
+unboxSingleTypeErr :: Untyped.T -> Untyped.T
```

Comment:
Why do these functions end in `Err`? Because we use `error` ? Should it be in a monad?

Reply:
yes if you call them wrong they error, but we should never get to that branch, #617 covers why I do this

Reply:
yup, makes sense, thanks